python_ta/transforms/setendings.py

```diff
@@ -430,6 +430,8 @@ def set_endings_from_source(node):
                 temp = node.end_col_offset
                 node.end_col_offset = j + 1
                 return
+            elif source_code[lineno][j] not in CONSUMABLES:
```

Comment:
Adding this line does cause the last failing test for Tuple node to now pass.
But it causes 6 other nodes to break. 

Pushed this change (lines 433-434) to this PR more to talk about the approach first.
I think this change should be reverted, and instead, a new transform function should be made for capturing ending `,` in Tuple node?